Source/Controllers/Preferences/Panes/SPFilePreferencePane.m

```diff
@@ -244,4 +332,24 @@ - (id)tableView:(NSTableView *)aTableView objectValueForTableColumn:(nullable NS
 	return [fileNames objectAtIndex:rowIndex];
 }
 
+- (void)tableView:(NSTableView *)tableView willDisplayCell:(id)cell forTableColumn:(NSTableColumn *)tableColumn row:(NSInteger)rowIndex{
```

Comment:
Potentially we could combine the bookmark and stale bookmark arrays and have isStale be an attribute of the bookmark instance? Might make some things a little simpler